frontend/src/Components/Error/Error.module.scss

```diff
@@ -0,0 +1,65 @@
+@import "~plaid-threads/scss/variables";
+.errorTop {
+  width: 90%;
+  height: 1px;
+  border-top: 1px solid $black200;
+}
+
+.errorContainer {
+  display: grid;
+  grid-template-columns: 15% 57% 28%;
+  width: 100%;
+  margin: 0;
+}
+
+.code {
+  margin: 2 * $unit 0px 2 * $unit 3 * $unit;
+  font-size: 1.4rem;
+}
+
+.errorContents {
+  margin: 3 * $unit 4 * $unit;
+  font-size: 1.2rem;
+}
+
+.errorItem {
+  display: grid;
+  grid-template-columns: 2fr 5fr;
+  margin-bottom: $unit;
+}
+
+.errorTitle {
+  font-weight: bold;
+  line-height: normal;
+}
+
+.errorData {
+  line-height: normal;
+  font-family: "Inconsolata";
+  letter-spacing: 0.25px;
```

Comment:
use threads typography? `<code>`

Reply:
the design has the font size smaller than `<code>`